app/routes/voyages.checkout.jsx

```diff
@@ -3,10 +3,10 @@ import PurchaseSummary from '../components/PurchaseSummary'
 
 const Checkout = () => {
   return (
-    <>
+    <div className="w-100% block mx-10 min-w-[860px] mt-12">
```

Comment:
Try to avoid hard-coded pixel values, as we'll need this to be responsive across many screen sizes.